src/test/groovy/org/prebid/server/functional/tests/OrtbConverterSpec.groovy

```diff
@@ -1277,9 +1307,137 @@ class OrtbConverterSpec extends BaseSpec {
+        def dur = PBSUtils.randomNumber
+        def slotinpod = PBSUtils.randomNumber
+        def apis = [PBSUtils.randomNumber]
+        def cat = [PBSUtils.randomString]
+        def cattax = PBSUtils.randomNumber
+        def bidResponse = BidResponse.getDefaultBidResponse(bidRequest).tap {
+            seatbid[0].bid[0].tap {
+                it.langb = langb
+                it.dur = dur
+                it.slotinpod = slotinpod
+                it.apis = apis
+                it.cat = cat
+                it.cattax = cattax
+            }
+        }
+
+        and: "Set bidder response"
+        bidder.setResponse(bidRequest.id, bidResponse)
+
+        when: "Requesting PBS auction with ortb 2.5"
+        def response = prebidServerServiceWithElderOrtb.sendAuctionRequest(bidRequest)
+
+        then: "Bidder request should contain seat[0].bid[0].{langb,dur,slotinpod,apis,cattax,cat} on request"
+        verifyAll(response.seatbid[0].bid[0]) {
+            it.langb == langb
+            it.dur == dur
+            it.slotinpod == slotinpod
+            it.apis == apis
+            it.cattax == cattax
+            it.cat == cat
+        }
+    }
+
+    def "PBS shouldn't remove saetbid[0].bid[].{lang,dur.slotinpod,apis,cat,cattax} when ortb request support 2.6"() {
+        given: "Default bid request with stored request object"
+        def storedRequestId = PBSUtils.randomString
+        def bidRequest = BidRequest.defaultBidRequest.tap {
+            ext.prebid.storedRequest = new PrebidStoredRequest(id: storedRequestId)
+        }
+
+        and: "Save storedRequest into DB"
+        def storedRequest = StoredRequest.getStoredRequest(bidRequest)
+        storedRequestDao.save(storedRequest)
+
+        and: "Default bidder response "
+        def langb = PBSUtils.randomString
+        def dur = PBSUtils.randomNumber
+        def slotinpod = PBSUtils.randomNumber
+        def apis = [PBSUtils.randomNumber]
+        def cat = [PBSUtils.randomString]
+        def cattax = PBSUtils.randomNumber
+        def bidResponse = BidResponse.getDefaultBidResponse(bidRequest).tap {
+            seatbid[0].bid[0].tap {
+                it.langb = langb
+                it.dur = dur
+                it.slotinpod = slotinpod
+                it.apis = apis
+                it.cattax = cattax
+                it.cat = cat
+            }
+        }
+
+        and: "Set bidder response"
+        bidder.setResponse(bidRequest.id, bidResponse)
+
+        when: "Requesting PBS auction with ortb 2.6"
+        def response = prebidServerServiceWithNewOrtb.sendAuctionRequest(bidRequest)
+
+        then: "Bidder request should contain seat[0].bid[0].{langb,dur,slotinpod,apis,cattax,cat} on request"
+        verifyAll(response.seatbid[0].bid[0]) {
+            it.langb == langb
+            it.dur == dur
+            it.slotinpod == slotinpod
+            it.apis == apis
+            it.cattax == cattax
+            it.cat == cat
```

Comment:
Maybe assert `response.seatbid == bidResponse.seatbid`?

Reply:
There are a lot different fields that really no need check